### DISCUSSION TITLE
React Server Side

### DISCUSSION THREAD

**Discussion by BAhmedDev** [community/community#111384] [Programming Help] [answered]
### Body

Can i run React on the server-side ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by SRayen** [comment, marked as answer]
React is a library designed for building user interfaces that run in the browser. However, you can achieve server-side rendering (SSR) functionality for React applications using frameworks like Next.js.

**Comment by arosha619** [comment]
Yes, you can run React on the server-side using frameworks like Next.js, which enables server-side rendering (SSR) for React applications.
